app/views/admin/sites/_form.html.haml

```diff
@@ -5,6 +5,7 @@
   = f.url_field :css_url
   = f.text_field :fqdn
   = f.text_field :tagline
+  = f.text_field :category_title_format
```

Comment:
How do you think about help text?
The user can't recognize what %{category_name}` is used as placeholder".